app/src/main/java/com/rahobbs/todo/fragments/DatePickerFragment.java

```diff
@@ -0,0 +1,79 @@
+package com.rahobbs.todo.fragments;
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.app.AlertDialog;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.DatePicker;
+
+import com.rahobbs.todo.R;
+
+import java.util.Calendar;
+import java.util.Date;
```

Comment:
`java.util.Date` is one of the worst classes EVER! If you don't mind adding a dependency I really recommend Joda date time. If dependencies are an issue / concern I get it and it's totally fine to use, but I think you'll find Joda is lot more readable / managable